gexpect_test.go

```diff
@@ -236,3 +236,43 @@ func TestRegexTimeoutWithOutput(t *testing.T) {
 		t.Fatalf("Didn't find %v in output: %v", searchPattern, out)
 	}
 }
+
+func TestRegexFindNoExcessBytes(t *testing.T) {
+	t.Logf("Testing Regular Expressions returning output with no excess strings")
+	repeats := 100
+	seqCmd := fmt.Sprintf("`seq 1 %d`", repeats)
```

Comment:
Can you also do a test with `seq -s ' ' 1 100`? I'd like to make sure that two matches on the same line work fine.